exporters/write_buffers/base_buffer.py

```diff
@@ -118,7 +118,8 @@ class GroupingBufferFilesTracker(object):
     that is cleaned up when calling close().
     """
 
-    def __init__(self, formatter, compression_format):
+    def __init__(self, formatter=None, compression_format=None, *args, **kwargs):
+        super(GroupingBufferFilesTracker, self).__init__(*args, **kwargs)
```

Comment:
http://stackoverflow.com/questions/8611712/what-does-objects-init-do-in-python

"super() does not always return a proxy for the parent class. Instead, it returns a proxy for the next class in MRO. In single-inheritance there is no difference between MRO and the inheritance chain. In multiple-inheritance, MRO may result in a class on the other inheritance chain instead."

It was needed to make diamond inheritance work.